scripts/qemu/auto_qemu_with_usb_storage

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+image_file_name="usbdisk.img"
+
+if [ -f "$file" ]
+then
+	echo "$image_file_name found."
+else
+	echo "$image_file_name not found."
+	qemu-img create -f qcow2 usbdisk.img 512M
```

Comment:
I'd suggest to add message about device creation here.

```
echo "$image_file_name not found. A new one will be created now..."
```

Reply:
Hm, if I run your script multiple times I got: `usbdisk.img not found` every time.

Reply:
Add `if [ -f $image_file_name ]`, see above. It fixes the problem.